[AN-167549] Update doc for carrier and A4T change (#47)
AdobeDocs/analytics-1.4-apis: docs/live-stream-api/metrics_dimensions.md

|browser|Browser as identified by the user agent string.|
|browserHeight|Height in pixels of browser window.|
|browserWidth|Width in pixels of browser window.|

> |carrier|The carrier of user’s cell phone. <br> Example: for mobile hits with valid IP, ```"carrier": "AT&T"```; for other hits, ```"carrier": ""```.|

Reviewer: ```suggestion
|carrier|The wireless provider for the mobile device. <br> Example: for mobile hits with valid IP, ```"carrier": "AT&T"```; for other hits, ```"carrier": ""```.|
```
